### ISSUE TITLE
ANO5 Gene has associated diseases in HPO database, but not in exomiser output

### ISSUE THREAD

**Issue by ccbruels** [exomiser/Exomiser#637]
Hi, we have 2 variants in the ANO5 gene that are P/LP, but the gene has an extremely low exomiser ranking. 

There are 7 diseases associated with ANO5 in the HPO terms (https://hpo.jax.org/browse/gene/NCBIGene:203859). 

The exomiser output shows no phenotype or disease associations for ANO5, which I think is the reason for the low exomiser rank.
[ANO5-NoDiseaseAssnExomiser.docx](https://github.com/user-attachments/files/26580216/ANO5-NoDiseaseAssnExomiser.docx)

The variants in ANO5 also have no disease ID or disease name in the exomiser output files variants.tsv or the vcf.gz.

137-1 
1381    11-22270378-G-C_AD      ANO5    102723370       AD      0.6511  0.0004  0.0000  0.9971  0.9971  1       0               rs760137559     11      22270378        22270378        G       C       0       50.3500 PASS    0|1     missense_variant        ANO5:ENST00000324559.9:c.1965G>C:p.(Trp655Cys)  UNCERTAIN_SIGNIFICANCE  PM5,PP3_Moderate,BP1                    523571  CONFLICTING_PATHOGENICITY_INTERPRETATIONS       1       0.86736 0.73    1.035   GNOMAD_G_NFE    0.0014699397    GNOMAD_E_NFE=8.992951E-5,GNOMAD_E_SAS=0.0011593128,GNOMAD_G_NFE=0.0014699397    ALPHA_MISSENSE  0.9973  REVEL=0.776,MVP=0.70669335,ALPHA_MISSENSE=0.9973

1467-1
1060    11-22257706-C-G_AD      ANO5    102723370       AD      0.5688  0.0010  0.0000  1.0000  1.0000  1       1               rs754889480     11      22257706        22257706        C       G       0       54.6600 PASS    1|0     stop_gained     ANO5:ENST00000324559.9:c.1359C>G:p.(Tyr453*)    UNCERTAIN_SIGNIFICANCE  PP5_Strong                      694040  PATHOGENIC      2       0.86736 0.73    1.035   GNOMAD_E_AMR    0.0022366361    GNOMAD_E_AMR=0.0022366361

The ANO5 gene is in the HPO terms included in the .yml files. The variants are only shown withe the FULL analysis option, and then filtering based on the gene name.

Is this an issue within exomiser?

**Comment by pnrobinson** [comment]
Are you able to share the HPO terms you used for the analysis or at least the count of terms and which of the three ANO5-associated diseases is deemed to be the correct diagnosis?

**Comment by ccbruels** [comment]
Hi,
We frequently don't have detailed histories available, so we start with
hpoIds: [ 'HP:0011805', 'HP:0011804' ] to cast a broader net, then narrow
the hpo list based on what we do know.

In these two specfic cases, we didn't see the clinically identified
variants (both had ANO5 hets, one also had a PYGM het) so we also tried
hpoIds: [ 'HP:0003560', 'HP:0003198' ] #md and myopathy and hpoIds: [
'HP:0001324' ] # muscle weakness.

Since we still weren't seeing the ANO5 or PYGM variants, for the full
analysis I used hpoIds: [ 'HP:0003560', 'HP:0003201', 'HP:0003202' ] # md,
elev ck, rhabdo.

My concern is that there are clearly related muscle diseases associated
with ANO5 , yet the exomiser disease fields were empty, likely leading to
the low exomiser score.

Best,
Christine

**Comment by pnrobinson** [comment]
Thanks!
In principle, those HPO terms should have at least some match (https://hpo.jax.org/browse/gene/NCBIGene:203859), and if the variant is either ClinVar P/LP or bioinformatics-pathogenic, then the disease should have a decent rank. @julesjacobsen any ideas how to troubleshoot this?

**Comment by julesjacobsen** [comment]
Are these two variants from two different cases? The variants are heterozygous whereas the muscular dystrophies linked to ANO5 in the Exomiser phenotype database (2512) are both recessive:

OMIM:611307	AR	Muscular dystrophy, limb-girdle, autosomal recessive 12
OMIM:613319	AR	Miyoshi muscular dystrophy 3
ORPHA:206549	AR	Anoctamin-5-related limb-girdle muscular dystrophy R12

They would both need to be present in the same case to fit an autosomal recessive compound heterozygous pattern. The only AD disorders are:

OMIM:166260	AD	Gnathodiaphyseal dysplasia
ORPHA:206599	AD	Isolated asymptomatic elevation of creatine phosphokinase
ORPHA:53697	AD	Gnathodiaphyseal dysplasia
 

Elevated circulating creatine kinase concentration is [HP:0003236](https://hpo.jax.org/browse/term/HP:0003236) not [HP:0003202](https://hpo.jax.org/browse/term/HP:0003202) (Skeletal muscle atrophy), so this would explain your lack of a good score. 

Using:
[ano5.vcf.txt](https://github.com/user-attachments/files/26724137/ano5.vcf.txt)
[ano5-phenopacket.yml.txt](https://github.com/user-attachments/files/26724192/ano5-phenopacket.yml.txt)

With:
```
java -Dexomiser.hg38.data-version=2512 -Dexomiser.phenotype.data-version=2512 -jar exomiser-cli-15.0.0.jar analyse --sample ano5-phenopacket.yml.txt --vcf ano5.vcf.txt --assembly hg38
```
Produces:
[ano5-exomiser.html](https://github.com/user-attachments/files/26724223/ano5-exomiser.html)

|Gene Symbol |	MOI |	Gene Identifier |	Gene Name |	Exomiser Score  | Phenotype Score |	Variant Score | 	Exomiser ACMG |	ClinVar ACMG |	
|-------------|-------------|-------------------|-------------------|--------------------|--------------------|----------------------|---------|----------|
|[ANO5](https://www.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000171714) 	| AD 	|[HGNC:27337](https://genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:27337) |	ENSG00000171714 |	0.841 p=3.0E-5 	|0.851 |	1.000 |	LIKELY PATHOGENIC |	PATHOGENIC|

What are the settings and data versions you're using for the analysis?

**Comment by ccbruels** [comment]
Hi Jules,

Thanks so much for your reply!

The two variants were from different cases. We are frequently looking for
additional variants that were missed on clinical testing, so it's helpful
to have hets scored even when the gene/disease is recessive.

You're correct about the HPO for elevated CK, I copied the wrong HPO into
my email, apologies.

I noticed that your command included exomiser-cli-15.1.0 jar, but the most
recent available version is 15.0.

When I ran:

java -Dexomiser.hg38.data-version=2512
-Dexomiser.phenotype.data-version=2512 -jar exomiser-cli-15.0.0.jar
analyse --sample ano5-phenopacket.yml.txt --vcf ano5.vcf.txt
--assembly hg38
I got the attached html that still shows the AD variant score of 1.0
but no ANO5 associated disorders or ANO5 phenotype matches.

I think my phenotype data is correct:
***@***.***:/projects/standard/pkang/shared/software/exomiser-cli-15.0.0$
ls -lt data/2512_phenotype/phenix/
total 11761
-rwxrwx---. 1 ccbruels pkang 5849718 Nov  5 04:28
ALL_SOURCES_ALL_FREQUENCIES_genes_to_phenotype.txt
-rwxrwx---. 1 ccbruels pkang 6192170 Nov  5 04:28 hp.obo
drwxrws---. 2 ccbruels pkang    4096 Nov  5 04:28 out
***@***.***:/projects/standard/pkang/shared/software/exomiser-cli-15.0.0$
ls -lt data/2512_phenotype/
total 21981230
-rwxrwx---. 1 ccbruels pkang 21757693952 Nov  6 07:36 2512_phenotype.mv.db
drwxrws---. 2 ccbruels pkang        4096 Nov  5 04:28 phenix
-rwxrwx---. 1 ccbruels pkang   740933632 Nov  5 04:28 rw_string_10.mv
-rwxrwx---. 1 ccbruels pkang    10151040 Nov  5 04:28 hp.obo

Is it possible this is due to a difference between v.15.0.0 and 15.1.0? If
so, when will 15.1.0 be available?

Thank you so much for your help!

[ano5.txt-exomiser.html](https://github.com/user-attachments/files/26800540/ano5.txt-exomiser.html)

Best,
Christine

**Comment by julesjacobsen** [comment]
Are you sure you're looking at the correct output file? It's nothing to do with a 15/15.1 difference.

15.1.0 will be out once we've finished testing, which might be a couple of weeks.

**Comment by ccbruels** [comment]
I ran this job thru our slurm scheduler, from the exomiser directory, with
the same command you used and the yml and vcf you posted (java
-Dexomiser.hg38.data-version=2512 -Dexomiser.phenotype.data-version=2512
-jar exomiser-cli-15.0.0.jar analyse --sample ano5-phenopacket.yml.txt
--vcf ano5.vcf.txt --assembly hg38). The output was in the exomiser results
directory, and I attached the html file previously.

AFAICT, I am looking at the correct output file. It is timestamped at the
time that the job completed, it's in the exomiser install results
directory, not our sample results directory tree. The settings show that it
used the expected vcf (from your previous post) and the same HPOs that you
used. Our HPC slurm scheduler is having a bad monday lol, so I am unable to
rerun that same analysis with the output going to a specified location and
name, but I'm as certain as I can be that I viewed and attached the correct
output file.

I can't see that there is anything different between our analyses, except
the exomiser version. Or possibly our data files didn't download correctly,
which is why I included a screenshot of the files and sizes on our hpc.

I checked a handful of genes in the full analysis that had a phenotype
score of 0.000, they had associated disorders and phenotype matches, but a
low score because they didn't match the input HPOs. Other genes are
annotated with associated disorders and phenotype matches.

I did notice that your output includes a Hugo gene name, while ours does
not. But all of the genes in our output don't have a Hugo gene name, ANO5
is the only one I noticed on a cursory review that does not have the
disorder or pheonotype matches.

Puzzling...

Best,
Christine

**Comment by julesjacobsen** [comment]
This is indeed puzzling. I'm not sure what to make of it. The only thing I can think of is that the application.properties might be pointing to an `exomiser.data-directory` where some different 2512 data resides, or your HPC was having an off day and you could try re-running things. The most likely explanation is that your 2512_phenotype data is corrupted, so you could try re-downloading and extracting.

**Comment by ccbruels** [comment]
Hi Jules,

I finally circled back to this.

With v15.0.0, I downloaded the phenotype data again as both
https://data.monarchinitiative.org/exomiser/data/2512_phenotype.zip and
https://g-879a9f.f5dc97.75bc.dn.glob.us/data/2512_phenotype.zip. I verified
that the application.properties was pointing to the correct
exomiser.data-directory,
For each download, I ran the command 2 ways just in case something was
specified incorrectly for our system. I ran java
-Dexomiser.hg38.data-version=2512 -Dexomiser.phenotype.data-version=2512
-jar exomiser-cli-15.0.0.jar analyse --sample ano5-phenopacket.yml.txt
--vcf ano5.vcf.txt --assembly hg38, and also
java -jar exomiser-cli-15.0.0.jar analyse --sample ano5-phenopacket.yml.txt
--vcf ano5.vcf.txt --assembly hg38 . Both had the same result, the variant
is annotated, has a variant score = 1 but phenotype score = 0 with no
phenotype matches or associated disorders.

I also did a clean install of v15.1.0. I only downloaded the data from
https://g-879a9f.f5dc97.75bc.dn.glob.us/data/2512_phenotype.zip (not the
monarch site). I verified that all data locations were correctly specified,
but had the same result.

Is it possible that both of the phenotype data files (
https://data.monarchinitiative.org/exomiser/data/2512_phenotype.zip and
https://g-879a9f.f5dc97.75bc.dn.glob.us/data/2512_phenotype.zip) have some
sort of error and that you are using a different version that does not have
an error?

I still haven't noticed any other genes having this issue. My concern is
that we will miss something important because I haven't noticed the lack of
phenotype matches. I don't have the tools to inspect the database files
directly.

Thanks so much for all of your help. AND for including the CADD phred
scores in the html file!
Best,
Christine

**Comment by ccbruels** [comment]
Hi,

Have you had a chance to look into the phenotype data files (https://data.monarchinitiative.org/exomiser/data/2512_phenotype.zip and
https://g-879a9f.f5dc97.75bc.dn.glob.us/data/2512_phenotype.zip) to see whether there is an issue with the files as posted? That seems the most likely source of this issue, since I've done clean installs of version 15.1.0 and the databases, and the error persists.

Thanks!
Christine

**Comment by akihikomit** [comment]
Hi,

Thank you for developing and maintaining Exomiser.

I encountered an unexpected filtering behavior for a symbolic structural
deletion in Exomiser 15.1.0.

Environment:
- Exomiser 15.1.0
- hg38
- Exomiser data release 2602
- analysisMode: FULL

The relevant PathogenicityFilter setting was:

pathogenicityFilter:
  keepNonPathogenic: false
  target: NON_CODING

The input VCF contains the following heterozygous deletion:

CHROM  POS      ID                       REF  ALT    END      SVTYPE  SVLEN  GT
chr19  2324116  Sniffles2.DEL.493S12    C    <DEL>  2324861  DEL     -745   0/1

Exomiser correctly annotates this variant as an exon-loss variant in LSM7:

Gene: LSM7
Entrez gene ID: 51690
Transcript: ENST00000252622.15
Functional class: exon_loss_variant
Gene phenotype score: 0.5355
Variant score: 1.0000

However, in the FULL output the variant has:

FILTER = path

and it is therefore absent from the original PASS_ONLY result.

This surprised me because the 15.1.0 release notes state that
target: NON_CODING should apply pathogenicity filtering to variants in
non-coding regions, whereas this SV is annotated by Exomiser itself as
exon_loss_variant.

In a more permissive rerun using:

pathogenicityFilter:
  keepNonPathogenic: true

the same deletion passes the PathogenicityFilter.

I realize that symbolic SVs may use a different annotation/scoring pathway
from SNVs/indels, so I may be misunderstanding the intended behavior.

Is it expected for a symbolic <DEL> annotated as exon_loss_variant to fail
PathogenicityFilter when target: NON_CODING is specified?

This may also be related to issue #638 concerning structural/non-coding
annotation pathways.

**Comment by akihikomit** [comment]
Hi,

I may have found a similar issue with SPG7 using Exomiser 15.1.0 / data 2602.

The FULL output shows:

GENE_SYMBOL = SPG7
ENTREZ_GENE_ID = 101930112
GENE_PHENO_SCORE = 0.0000

However, the canonical NCBI Gene ID for SPG7 is 6687.

Could this incorrect/ambiguous gene-ID mapping explain the phenotype score of 0?

Thanks.
